### ISSUE TITLE
scrcpy-linux-v3.0.tar.gz is not gzipped as claimed by the file name

### ISSUE THREAD

**Issue by sedrik** [Genymobile/scrcpy#5581]
## Environment

 - **OS:** Ubuntu 24.04
 - **Scrcpy version:** 3.0
 - **Installation method:** github release download
 - **Device model:**
 - **Android version:** not applicable

## Describe the bug

Based on the `tar.gz` file ending it is assumed that the tarball is gzipped, but it turns out that it is not as evident by the output bellow.

```
~/Downloads
❯ sha256sum scrcpy-linux-v3.0.tar.gz
06cb74e22f758228c944cea048b78e42b2925c2affe2b5aca901cfd6a649e503  scrcpy-linux-v3.0.tar.gz

~/Downloads
❯ cat SHA256SUMS.txt
800044c62a94d5fc16f5ab9c86d45b1050eae3eb436514d1b0d2fe2646b894ea  scrcpy-server-v3.0
06cb74e22f758228c944cea048b78e42b2925c2affe2b5aca901cfd6a649e503  scrcpy-linux-v3.0.tar.gz
7cbf8d7a6ebfdca7b3b161e29a481c11088305f3e0a89d28e8e62f70c7bd0028  scrcpy-win32-v3.0.zip
dfbe8a8fef6535197acc506936bfd59d0aa0427e9b44fb2e5c550eae642f72be  scrcpy-win64-v3.0.zip
5db9821918537eb3aaf0333cdd05baf85babdd851972d5f1b71f86da0530b4bf  scrcpy-macos-v3.0.tar.gz

~/Downloads
❯ tar -xvzf scrcpy-linux-v3.0.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

~/Downloads
❯ tar -xvf scrcpy-linux-v3.0.tar.gz
scrcpy-linux-v3.0/
scrcpy-linux-v3.0/scrcpy-server
scrcpy-linux-v3.0/icon.png
scrcpy-linux-v3.0/scrcpy
scrcpy-linux-v3.0/adb
scrcpy-linux-v3.0/scrcpy.1
scrcpy-linux-v3.0/scrcpy_bin
```

**Comment by rom1v** [comment]
Oh, thank you. Good catch!

I thought that `tar` inferred the compression from the file extension when the flag was not explicit, that's not true.

Fixed by 36574d2ee7b5084a64c54951e7c97f5a46ed0388 :rocket:

**Comment by sedrik** [comment]
Thanks for the quick fix and an awesome project <3
